lib/please_cli/please_cli/config.py

```diff
@@ -159,11 +167,14 @@
                 'heroku_app': 'releng-staging-clobberer',
                 'heroku_dyno_type': 'web',
                 'url': 'https://clobberer.staging.mozilla-releng.net',
+                # TODO: do staging and production have the same dns?
```

Comment:
@dividehex Looking at [route53.tf](https://github.com/mozilla-releng/build-cloud-tools/blob/master/terraform/base/route53.tf) it looks like production and staging use the same record. is this intentional or a typo?